### PR TITLE
Updating documentation regarding nvm in Dockerfile.

### PR DESCRIPTION
**Pull request by charginghawk**
https://github.com/docksal/docksal/issues/798#issuecomment-429045813

### CODE REVIEW

File: docs/content/stack/extend-images.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ USER docker
 # Install additional global npm dependencies
 RUN \
 	# Initialize nvm environment
```
Comment (lmakarov) [inline docs/content/stack/extend-images.md]:
```suggestion
	# Initialize the user environment (this loads nvm)
```

File: docs/content/stack/extend-images.md [hunk 2]
```diff
@@ -159,4 +159,4 @@ image, submit a pull request.
 - In the Linked Repositories section, specify `docksal/<service>`. This way, every time Docksal updates that service image 
 on Docker Hub, your image will automatically be rebuilt as well based on the latest changes.
 
-See the example use case for [maintaining a custom docker image](/use-cases/docker-image).
\ No newline at end of file
+See the example use case for [maintaining a custom docker image](/use-cases/docker-image).
```
Comment (lmakarov) [inline docs/content/stack/extend-images.md]: What's the change here?

Comment (achekulaev) [reply docs/content/stack/extend-images.md]: line ending

File: docs/content/stack/extend-images.md (hunk 1 above)
Comment (lmakarov) [reply docs/content/stack/extend-images.md]: Ping @charginghawk. Please accept the suggested change, so we can merge this PR and get you a commit credit :)

**Pull request merged**